src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellSectionRootRenderer.cs

```diff
@@ -255,7 +255,7 @@ protected virtual void LoadRenderers()
 
 				if (item == currentItem)
 				{
-					_containerArea.AddSubview(renderer.PlatformView);
+					_containerArea.AddSubview(renderer.ViewController.View);
```

Comment:
Here's the only additional change from the original PR https://github.com/dotnet/maui/pull/13332